modules/op-release-notes-1-20-0.adoc

```diff
@@ -0,0 +1,289 @@
+
+New buildah-ns task for improved container build security::
+* {pipelines-shortname} adds the `buildah-ns` task. It improves container build security with user namespace isolation and remains compatible with the existing `buildah` task. For more details, see "Additional resources".
+
+readOnlyRootFilesystem enabled for default deployments::
+* Containers in the {pipelines-shortname}, {tekton-results}, {tekton-chains}, and Manual Approval Gate deployments have `readOnlyRootFilesystem` setting enabled by default, improving security and compliance.
+
+Tasks display friendly names in the web console::
+* Tasks in the {OCP} web console now use the displayName property instead of technical task names, improving readability and making tasks easier to search within the {pipelines-title} interface.
+
+{pipelines-shortname} on single-node OpenShift::
+* Tech preview support for running {pipelines-shortname} on Single-Node OpenShift (SNO) is now available. This support is not intended for production deployments. {pipelines-shortname} on SNO has no additional limitations beyond the inherent constraints of a single-node cluster: limited scalability, no redundancy, and constrained concurrency.
+For a cluster on libvirt UPI, the minimum hardware requirements are:
+
+** 12 vCPUs
+** 64 GB RAM
+** 240 GB disk space
+
+:FeatureName: {pipelines-shortname} on Single-Node OpenShift (SNO)
+include::snippets/technology-preview.adoc[]
+
+.Operator
+
+Independent control of RBAC and CA bundle config map creation::
+* With this update, you can independently control whether the {pipelines-shortname} Operator creates Role-Based Access Control (RBAC) resources and Trusted Certificate Authority (CA) bundle config maps in your cluster. This enhancement gives you more flexibility to fit your environment and helps avoid resource duplication. By default, both options are enabled to maintain the existing behavior. For more details, see "Additional resources".
+
+.Pruner
+
+Event-based pruner configurable in TektonConfig CR::
+* You can enable and configure the event-based `tektonpruner` directly in the `TektonConfig` custom resource (CR). Observability enhancement is also included with the new metrics specific to pruner. 
+
+:FeatureName: The event-based pruner
+include::snippets/technology-preview.adoc[]
+
+.{tekton-triggers}
+
+Optional installation of {tekton-triggers} through the Operator::
+* You can choose to not install {tekton-triggers} when deploying {pipelines-shortname} through the Operator. This enhancement provides finer control over component installation, allowing flexibility for environments where triggers are managed separately. The default setting is `false`.
```

Comment:
When deploying {pipelines-shortname} through the Operator, you can choose not to install {tekton-triggers}. This update provides finer control over component installation and supports environments where triggers are managed independently. The default value is `false`.